### ISSUE TITLE
Plans for ARC (Authenticated Received Chain)?

### ISSUE THREAD

**Issue by wioxjk** [Pro/dkim-exchange#203]
Are there any plans to support ARC with this plugin?

**Comment by Pro** [comment]
No, unfortunately I'm very time-limited. But if you have time feel free to submit a Pull Request

**Comment by jstedfast** [comment]
I've started to implement ARC support in [MimeKit](https://github.com/jstedfast/MimeKit).

You can keep tabs on my progress by subscribing to https://github.com/jstedfast/MimeKit/issues/386
